vendor/github.com/openshift/api/operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml

```diff
@@ -51,7 +51,6 @@ spec:
                 - csi.sharedresource.openshift.io
                 - diskplugin.csi.alibabacloud.com
                 - vpc.block.csi.ibm.io
-                - powervs.csi.ibm.com
```

Comment:
@Madhan-SWE @mkumatag The vendor update for this API concerns me a little. Is it safe to remove these, or does the openshift API need updating?

Reply:
All the values for the enum is still there, just that line nums got changed a a little . And `filestore.csi.storage.gke.io` got added newly.